firefox-ios/Shared/Strings.swift

```diff
@@ -5670,6 +5670,41 @@ extension String {
         tableName: nil,
         value: "Gives websites a dark appearance. Some sites might not look right.",
         comment: "Under Website Appearance section in Appearance menu, this is the text under the toggle describing the dark mode feature.")
+    public static let PageZoomSectionHeader = MZLocalizedString(
```

Comment:
These seem ripe for being in a Settings struct of sorts and organizing that. Also, these should have a `tableName`, in case we need to experiment.